modules/nf-core/cellranger/mkfastq/tests/main.nf.test

```diff
@@ -71,33 +71,5 @@ nextflow_process {
                 }
             )
         }
-
-    }
-
-    test("cellranger - tiny - simple - stub") {
```

Comment:
had to remove this nf-test because it doesn't produce reads, only undetermined

Reply:
I don't understand what do you mean, I can update the test instead of removing it so that it checks the process.out.undetermined_fastq instead, but the process.out.fastq is empty for this removed test

Reply:
Added the stub test, thanks!